src/interface/interface.ts

```diff
@@ -179,6 +192,7 @@ export interface Image {
   description_url: string;
   avator_url: string;
   circuit_size: number;
+  context?: Uint8Array;
```

Comment:
If this one should be as same as rust's image struct, it need add both init_context and context?